operators/derive/to_mapbox.py

```diff
@@ -10,54 +9,52 @@
 
 def geo_data_flow():
     return DF.Flow(
-        DF.load(f'{settings.DATA_DUMP_DIR}/flat_table/datapackage.json'),
+        DF.load(f'{settings.DATA_DUMP_DIR}/card_data/datapackage.json'),
         DF.update_package(name='Geo Data'),
-        DF.update_resource(['flat_table'], name='geo_data', path='geo_data.csv'),
+        DF.update_resource(['card_data'], name='geo_data', path='geo_data.csv'),
+        DF.add_field(
+            'record',
+            'object',
+            lambda r: {k: str(v) for k, v in r.items()},
+            resources=['geo_data'],
+        ),
+        DF.add_field(
+            'offset',
+            'array',
+            lambda r: (0, 0),
+            constraints={'maxLength': 2},
+            resources=['geo_data'],
+        ),
+        helpers.unwind('response_categories', 'response_category'),
         # some addresses not resolved to points, and thus they are not useful for the map.
         DF.filter_rows(lambda r: not r['branch_geometry'] is None, resources=['geo_data']),
         DF.join_with_self(
             'geo_data',
-            ['service_id', 'response_id', 'branch_id'],
+            ['card_id'],
```

Comment:
I think this will just choose one response category per card (there can be more than one)

Reply:
I think we should probably have `geometry` and `response_category` as join key here?

Reply:
yes